app/controllers/email_subscriptions_controller.rb

```diff
@@ -1,28 +1,25 @@
 class EmailSubscriptionsController < ApplicationController
-  # No need to authorize this because its implicit when unsubbing
+  PREFERRED_EMAIL_NAME = {
```

Comment:
Given that I recently introduced a similar lookup hash, do you think it would make sense to extract this into a module? Something like:

```ruby
module Email
  PREFERRED_NAME = {
    ...
  }

  LIST_MAPPINGS = {
    ...
  }
end
```

Reply:
I think if there's another use this in a different class I'll definitely extract it